truffle.js

```diff
@@ -64,6 +64,20 @@ module.exports = {
             },
             network_id: 4,
             gas: 6600000
+        },
+        rinkebyDryRun: {
```

Comment:
Added this network config which can be passed in when we specifically want to do a dry run on Rinkeby that mimics what the deployment would look like on main net i.e. no faucet